pkg/poll/poll_test.go

```diff
@@ -282,6 +258,64 @@ var _ = Context("Waiting for resource", func() {
 	)
 })
 
+var _ = Context("Waiting for Build", func() {
+	When("resource is not created yet", func() {
```

Comment:
I tough we agreed not to use `When` statements in ginkgo.
@qbarrand do you remember what were the guidlines we agreed on?
I think it was `Describe` for pkg, `Context` for function and `It` for all the cases we want to test in a specific function.

Please look at the current test to keep consistent with what we already have.

Reply:
Is there any place where I can check the guidelines?